src/components/App.js

```diff
@@ -1,12 +1,26 @@
-import React from 'react';
-import '../styles/App.css';
+import React, { useState } from "react";
+import Search from "./Search";
+import SearchResults from "./SearchResults";
+
+import "../styles/App.css";
+
+const App = () => {
+  const [searchResults, setSearchResults] = useState([]);
 
-function App() {
   return (
     <div className="App">
-      <p>Hello World</p>
+      <img
+        className="nasa-logo"
+        src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
+        alt="nasalogo"
+      />
+      <Search setSearchResults={setSearchResults} />
+
+      {searchResults &&
+        searchResults.length > 0 &&
```

Comment:
Good checking. You could get rid of the > 0 here as 0 is falsy. You could also make a bookean called showResults outside of your return if you wanted to not have both the checks here. There is a lodash function called get that makes nested checking if things exist https://github.com/lodash/lodash/blob/master/get.js if you wanted to check it out :)
Lots of options, but the code works as intended so not a necessary change for functionality.